test/verify/check-system-services

```diff
@@ -269,6 +269,10 @@ Unit=test.service
         b.enter_page("/system/services")
         b.wait_js_cond('window.location.hash === "#/test.service"')
 
+        # Check that current memory is shown
+        b.wait_visible("#memory")
+        self.assertGreater(int(b.text("#memory").split(" ")[0]), 0)
```

Comment:
Please 
 - check that there is no Listen
 - stopping the unit makes Memory go away, starting it again brings it back
 - ideally it should also test that the regular update works. Schema: Let test.service wait for a /tmp/do-memhog file, then have it allocate a big chunk (see check-metrics for existing code); the test then gets to this point, creates do-memhog, and ensures that the shown value actually increases (it should take at most 30s, so our standard timeouts seem fine)